src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -330,7 +330,7 @@ public function parsePostfixExpression($node)
                     throw new SyntaxError('Expected name', $token->cursor);
                 }
 
-                $arg = new Node\ConstantNode($token->value);
+                $arg = new Node\NameNode($token->value);
```

Comment:
I tend to agree that this semantically more correct. Is it a BC break? I don't think so.